e2e-tests/portal/cypress/e2e/journeys/maker/fill-and-submit-issue-facility-form/fillAndSubmitIssueBondFacilityForm.js

```diff
@@ -1,46 +1,26 @@
 const pages = require('../../../pages');
-const dateConstants = require('../../../../../../e2e-fixtures/dateConstants');
+const { oneMonth, today, twoDays } = require('../../../../../../e2e-fixtures/dateConstants');
 
-export const ISSUED_BOND_DATE_VALUE = dateConstants.today;
-export const COVER_START_DATE_VALUE = dateConstants.twoDays;
-export const COVER_END_DATE_VALUE = dateConstants.oneMonth;
+export const ISSUED_BOND_DATE_VALUE = today;
+export const COVER_START_DATE_VALUE = twoDays;
+export const COVER_END_DATE_VALUE = oneMonth;
 
 export const fillAndSubmitIssueBondFacilityFormWithoutRequestedCoverStartDate = () => {
-  cy.keyboardInput(pages.bondIssueFacility.issuedDateDayInput(), dateConstants.todayDay);
-  cy.keyboardInput(pages.bondIssueFacility.issuedDateMonthInput(), dateConstants.todayMonth);
-  cy.keyboardInput(pages.bondIssueFacility.issuedDateYearInput(), dateConstants.todayYear);
+  cy.completeDateFormFields({ idPrefix: 'issuedDate' });
 
-  cy.keyboardInput(pages.bondIssueFacility.coverEndDateDayInput(), dateConstants.oneMonthDay);
-  cy.keyboardInput(pages.bondIssueFacility.coverEndDateMonthInput(), dateConstants.oneMonthMonth);
-  cy.keyboardInput(pages.bondIssueFacility.coverEndDateYearInput(), dateConstants.oneMonthYear);
+  cy.completeDateFormFields({ idPrefix: 'coverEndDate', date: oneMonth });
```

Comment:
It's not recognising the cypress command here - is there somewhere else it needs to be added? 
![image](https://github.com/user-attachments/assets/fd72455f-33d3-403b-afcd-a9d8653bd177)

Reply:
(same in other file in this directory)

Reply:
Thank you, this is in the UKEF E2E suite - have added.